### ISSUE TITLE
Special Move Editor

### ISSUE THREAD

**Issue by goblinJoel** [rpgtoolkit/editor#22]
The special move editor was implemented and working as of my commit logs from awhile ago. This issue is to help us keep track of what is and isn't done.

**Comment by goblinJoel** [comment]
The special move editor is open to enhancement suggestions.

**Comment by swordmaster2k** [comment]
Marking this as out of scope for 4.0.

**Comment by goblinJoel** [comment]
Alright. Unless it regressed though, it was working. Did we need to redo something about it?

**Comment by swordmaster2k** [comment]
We'll probably just need to review it after we've established a new battle system, seeing how the current one isn't being carried over in any form.

Doubt I'll get a full default BS before 4.0, only an example :(

**Comment by goblinJoel** [comment]
Makes sense. So, for this release, we can just leave the code there, but remove the special moves editor from the interface and remove special moves from the list of file types the file dialog recognizes, right?

I don't think it's necessarily bad to have the editor done with only an example BS. It gives us parameters to work with in designing a default BS. (Obviously, this could go either way. If the engine had a default BS all complete, we could have built the editor around that.)
